requirements.txt

```diff
@@ -0,0 +1,3 @@
+gdom
+jq
```

Comment:
AFAIK jq isn't called from Python right now

Reply:
gdom neither, what prevents us form moving both to `frontend.py`?

Reply:
We don't know how to call gdom as a library from Python yet.

Reply:
Something like https://stackoverflow.com/a/89243/4682621?

Reply:
Would've hoped not to have to do it this way :/
Yeah but maybe that's the simplest way.

Reply:
I'll give it a shot